builtin/common/tests/misc_helpers_spec.lua

```diff
@@ -167,6 +167,11 @@ describe("table", function()
 		assert.equal(1, table.indexof({"foo", "bar"}, "foo"))
 		assert.equal(-1, table.indexof({"foo", "bar"}, "baz"))
```

Comment:
general note: it would make sense to extend these with failure cases in case of holes and duplicate values

Reply:
Done in b0aba169d

Reply:
wouldn't this make more sense as part of #14909 ?